src/content/pages/index.mdx

```diff
@@ -111,32 +113,26 @@ Because Mic Drop mutes your audio at the system level, it works with any app you
 
 </Panel>
 
-
-
-<TestimonialCarousel />
-
-
-<div css={css`padding-top: 0.8rem;`}>
-<Panel gridTablet="1fr 48rem 1fr">
-
-<Spacer />
+<Panel>
 
 <TextBlock>
 
-## Get ready for your meetings to suck less.
+## Your purchase supports charity
 
-We can't promise to stop your overly-enthusiastic co-worker from cutting you off, but we _can_ make it a little bit easier to control your _own_ microphone.
+We started building Mic Drop as a tool for remote workers before the COVID-19 pandemic. In addition to _working_ from home, lockdown measures mean people who experience domestic abuse are at greater risk.
```

Comment:
I tightened this up and removed the link. It was pretty distracting/wordy… but worse—I think it was a bit too glum and not inspiring. I know that's a pretty delicate balance to strike, but I don't think the original one did the trick. 😓

If this is okay let's go with that, thoughts?

(Also I'm happy not to link to the Guardian. 😆)